frontend/src/components/layouts/components/Content.tsx

```diff
@@ -0,0 +1,15 @@
+import { Layout } from 'antd';
+import styled from 'styled-components';
+
+import { Colors } from '../../../styles';
+
+const { Content: AContent } = Layout;
+
+const Content = styled(AContent)`
+  background-color: ${Colors.onPrimary};
+  box-shadow: 0px 5px 30px -6px rgba(100, 100, 100, 0.5);
```

Comment:
IMO Komponent `Content` nie powinien mieć box-shadow. Jest to element, który będzie wypełniał całą stronę. Tutaj tak właściwie wystarczy, że wrzucić samo stylowanie tła.